app/views/idv/review/new.html.erb

```diff
@@ -36,9 +36,13 @@
                phone: PhoneFormatter.format(@applicant[:phone]) %>
   <% end %>
 
-  <%= f.button :submit, t('forms.buttons.continue'), class: 'usa-button--big usa-button--wide' %>
+  <%= f.button(
+    :submit,
+    t('forms.buttons.continue'),
+    class: 'usa-button--big usa-button--wide margin-top-4',
+  ) %>
```

Comment:
can we indent past the `<%` ?

```suggestion
  <%= f.button(
        :submit,
        t('forms.buttons.continue'),
        class: 'usa-button--big usa-button--wide margin-top-4',
      ) %>
```

Reply:
To me, this feels similar to what I'd hoped we'd done away with in #4919 as far as parameter alignment, but not something I feel especially strong about if we prefer this format. But there's a lot of inconsistency already, so it'd at least be nice to enforce this one way or the other. @mitchellhenke and I had been looking at the [Rubocop feature of ERBLint](https://github.com/Shopify/erb-lint#Rubocop), which could potentially help here? I haven't tested.

Reply:
I'd definitely be open to that!

Reply:
I tinkered locally a bit with Rubocop + ERBLint just to see if either of the above formats seem to work well with it. The suggested revision seemed to be the ticket, as the current implementation gives...

```
Layout/FirstArgumentIndentation: Indent the first argument one step more than the start of the previous line.
In file: app/views/idv/review/new.html.erb:40
```